### PR TITLE
Sign out error on logout

### PR DESCRIPTION
**Pull request by oleggnidets-okta**
### Problem Analysis (Technical)
There's an error returned that refresh token cannot be revoked on sign out. It occurs even if the tokens doesn't exists. In admin panel the app can have omitted Refresh Token. 
Scope of [OKTA-380078](https://oktainc.atlassian.net/browse/OKTA-380078)

### Solution (Technical)
Added check on refresh token existence. 

### Affected Components
`OktaOidcSignOutHandler`

### Tests
Added.

### CODE REVIEW

File: Example/ViewController.swift [hunk 1]
```diff
@@ -146,12 +146,15 @@ final class ViewController: UIViewController {
...
+            switch currentOption {
+            case .revokeAccessToken, .revokeRefreshToken, .removeTokensFromStorage, .revokeTokensOptions:
                 self.updateUI(updateText: "Revoking tokens...")
-            } else if currentOption.contains(.revokeRefreshToken) {
-                self.updateUI(updateText: "Revoking tokens...")
-            } else if currentOption.contains(.signOutFromOkta) {
+            case .signOutFromOkta:
                 self.updateUI(updateText: "Signing out from Okta...")
+            case .allOptions:
+                break
```
Comment (anastasiiaiurok-okta) [inline Example/ViewController.swift:155]: Why do we need to handle `.allOptions` case? The behaviour is the same as for `default`

**Review by anastasiiaiurok-okta — APPROVED** [review]

File: Example/ViewController.swift (hunk 1 above)
Comment (oleggnidets-okta) [reply Example/ViewController.swift:155]: Not really, if a new case will be added to enum, the compiler suggests to fulfill `switch case`. 

Also, it's a good practice to make switch exhaustive to show cases which are not handled.

Comment (oleggnidets-okta) [reply Example/ViewController.swift]: Done)

**Review by IldarAbdullin-okta — APPROVED** [review]
LGTM! 👍

**Pull request merged**